.github/workflows/test.yml

```diff
@@ -35,8 +35,11 @@ jobs:
           bash ./make_certs.sh
           cd ..
 
+      - name: Start MQ and DB
+        run: docker-compose -f dev_utils/compose-no-tls.yml up -d db mq
+
       - name: Test
-        run: go test -v -coverprofile=coverage.txt -covermode=atomic ./...
+        run: go test --tags=integration -v -coverprofile=coverage.txt -covermode=atomic ./...
```

Comment:
is this tag needed?

Reply:
Yes, otherwise the tests against the live MQ server will not run.

Reply:
Yeah, figured it had something to do with that.  In my local ubuntu both tests run regardless of `--tags` but maybe this works differently in actions.